### PR TITLE
Bug 1930184: Fix formatting and description for WMCO 2.0.0

### PR DESCRIPTION
**Pull request by mansikulkarni96**
This PR corrects the CSV description to mention pre-requisite of 4.7 clusters and
formats the VM template requirements.

### CODE REVIEW

File: deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ spec:
 
     ### Pre-requisites
     * A Red Hat OpenShift subscription
-    * OCP 4.6.8+ cluster running on Azure, AWS or vSphere configured with hybrid OVN Kubernetes networking
+    * OKD/OCP 4.7 cluster running on Azure, AWS or vSphere configured with hybrid OVN Kubernetes networking
```
Comment (aravindhp) [inline deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml:36]: Product WMCO is only for OCP. So `OCP 4.7` is sufficient.

**Review by aravindhp — CHANGES_REQUESTED** [review]
Thanks for fixing this, @mansikulkarni96. Please address my comment.

**Comment by codyhoag** [comment]
LGTM!

File: deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (mansikulkarni96) [reply deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml]: fixed

**Review by aravindhp — APPROVED** [review]
/approve

**Comment by aravindhp** [comment]
/retest

```
         --- FAIL: TestWMCO/create/Metrics_validation (170.57s)
            --- FAIL: TestWMCO/create/Metrics_validation/Windows_Exporter_configuration_validation (140.45s)
                --- PASS: TestWMCO/create/Metrics_validation/Windows_Exporter_configuration_validation/e2e-wmcow-dcwp5 (5.05s)
                --- FAIL: TestWMCO/create/Metrics_validation/Windows_Exporter_configuration_validation/e2e-wmcow-vhs5j (135.38s) 
```
Given that it passed for one node and not the other, I suspect throttling.

**Pull request merged**